parcels/examples/tutorial_delaystart.ipynb

```diff
@@ -42,446 +30,848 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
    "source": [
-    "Define an empty `ParticleSet`"
+    "### Assigning each particle its own `time` ###"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The simplest way to delaye the start of a particle is to use the `time` argument for each particle"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 3,
-   "metadata": {
-    "collapsed": true,
-    "deletable": true,
-    "editable": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "pset = ParticleSet(fieldset=fieldset, lon=[], lat=[], pclass=JITParticle)"
+    "npart = 10  # number of particles to be released\n",
+    "lon = 0.025 * np.ones(npart)\n",
+    "lat = np.linspace(0.025 , 0.4, npart, dtype=np.float32)\n",
+    "time = np.arange(0, npart) * delta(hours=1).total_seconds()  # release every particle one hour later\n",
+    "\n",
+    "pset = ParticleSet(fieldset=fieldset, pclass=JITParticle, lon=lon, lat=lat, time=time)"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "deletable": true,
-    "editable": true
-   },
+   "metadata": {},
    "source": [
-    "And now also define some other variables"
+    "Then we can execute the `pset` as usual"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 4,
-   "metadata": {
-    "collapsed": true,
-    "deletable": true,
-    "editable": true
-   },
-   "outputs": [],
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO: Compiled JITParticleAdvectionRK4 ==> /var/folders/r2/8593q8z93kd7t4j9kbb_f7p00000gn/T/parcels-501/4b31da299158ef673e01491b786d6e1b.so\n"
+     ]
+    }
+   ],
    "source": [
-    "npart = 10  # number of particles to be released\n",
-    "delaytime = delta(hours=1)  # time between each particle release\n",
-    "endtime = delta(hours=24)  # total time of the experiment"
+    "output_file = pset.ParticleFile(name=\"DelayParticle_time\")\n",
+    "pset.execute(AdvectionRK4, runtime=delta(hours=24), dt=delta(minutes=5),\n",
+    "             interval=delta(hours=1), output_file=output_file)"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "deletable": true,
-    "editable": true
-   },
+   "metadata": {},
    "source": [
-    "We use the same start locations as in `examples/example_peninsula.py`"
+    "And then finally, we can show a movie of the particles. Note that the southern-most particles start to move first."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 5,
-   "metadata": {
-    "collapsed": true,
-    "deletable": true,
-    "editable": true
-   },
-   "outputs": [],
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
```

Comment:
Why do we have to commits pynb with the outputs inside? It makes the diff complicate to read, and add unnecessary lines with no meaning. Or do you need it in order to visualise something on oceanparcels.org?
This file is really thousands of lines!

Reply:
I know it's inconvenient that these files are so massive. But we need to save them with all these outputs to be displayed properly on nbviewer.jupyter.org (where we link to from the oceanparcels.org tutorial list); so nothing we can do bout that, I fear